app/views/dashboard/_navigation.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= render TabsComponent.new do |component| %>
+  <%= component.tab(path: dashboard_path) do %>
+    <strong>Overview</strong>
+  <% end %>
+  <%= component.tab(path: start_lead_providers_report_schools_path) do %>
+    <strong>Schools</strong>
+  <% end %>
+<% end %>
```

Comment:
I might have added the same navigation inline in the report page. Could you reuse this version there?